docs/src/using-turing/advanced.md

```diff
@@ -128,16 +128,18 @@ mf(vi, sampler, ctx, model) = begin
     )
 
     # Add the lp to the accumulated logp.
-    vi.logp += lp
+    acclogp!(vi, lp)
 
     # Observe each value of x[i], according to a
     # Normal distribution.
-    vi.logp += Turing.Inference.dot_tilde(
-        ctx, 
-        sampler, 
-        Normal(m, sqrt(s)), 
-        x, 
-        vi,
+    acclogp!(vi,
+        Turing.Inference.dot_tilde(
+            ctx, 
+            sampler, 
+            Normal(m, sqrt(s)), 
+            x, 
+            vi,
+        ),
```

Comment:
Maybe split this into two steps, i.e., `lp = ....; acclogp!(vi, lp)` as above? IMO that is easier to read.

Reply:
Fair enough.